enterprise/static/enterprise/admin/enterprise_customer_catalog.js

```diff
@@ -0,0 +1,38 @@
+(function($){
+    $(document).ready(function(){
+        var catalogQueryId,
+            ignoreUpdate = true,
+            catalogQueryApiUrl = '/enterprise/api/v1/enterprise_catalog_query/',
+            $contentFilterEl = $('#id_content_filter');
+
+        $("[name='enterprise_catalog_query']").on("change", function(event){
+            catalogQueryId = $(event.target).val();
+            if (!ignoreUpdate && catalogQueryId) {
+                // for some reasons $.cookie is not defined in this scope.
+                function readCookie(name) {
```

Comment:
Since we don't need to get csrftoken, we don't need to read cookie. So this can be removed as well.